src/components/Button/V2.jsx

```diff
@@ -4,43 +4,31 @@ import PropTypes from 'prop-types'
 import { fontSize, devices } from '../theme'
 
 const ButtonV2 = styled.button`
-  text-align: center;
-  display: flex;
-  align-items: center;
-  justify-content: center;
   cursor: pointer;
-
+  display: inline-flex;
+  align-items: center;
+  gap: 0.75rem;
+  padding: 0.5em 0.75em;
```

Comment:
I think removing the `text-align` property made the button use in other places weird looking?

From the wallet landing screen:

![image](https://user-images.githubusercontent.com/12353734/150552692-3f2949a4-b125-4bb0-9f19-152515752e16.png)

Reply:
Good call! Hadn't noticed that button V2 was used in this context, will correct this